README.md

```diff
@@ -414,12 +414,34 @@ were necessary.
 Each commit should ideally contain changes that are functionally
 connected/related.  For example, changes to the command line parsing
 code, changes to the documentation, and changes to some unrelated
-comments may be split into three commits.
+comments may be split into three commits. Also note that a single logical
```

Comment:
Would it be beneficial to add a line related to #20, e.g. "A rule of thumb is that if the commit message contain an 'and', it should probably be split into two commits"? Or perhaps superfluous?

Reply:
can you suggest a change like @viklund did ?

Reply:
```suggestion
comments may be split into three commits. A rule of thumb is that if the commit message contains an 'and', it should probably be split into multiple commits. Also note that a single logical
```